component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPTokenGenerator.java

```diff
@@ -69,285 +72,360 @@
  */
 public class TOTPTokenGenerator {
 
-	private static final String FIRST_NAME = "firstname";
-	private static final String TOTP_TOKEN = "totp-token";
-	private static final Log log = LogFactory.getLog(TOTPTokenGenerator.class);
+    private static final String FIRST_NAME = "firstname";
+    private static final String TOTP_TOKEN = "totp-token";
+    private static final Log log = LogFactory.getLog(TOTPTokenGenerator.class);
 
-	private static final int TOKEN_HASH_DIVISOR = 1000000;
-	// Max number of attempts to calculate a token with minimum chars.
-	private static final int MAX_TOKEN_CALCULATE_ATTEMPTS = 5;
+    private static final int TOKEN_HASH_DIVISOR = 1000000;
+    // Max number of attempts to calculate a token with minimum chars.
+    private static final int MAX_TOKEN_CALCULATE_ATTEMPTS = 5;
 
-	/**
-	 * Get Time steps from unix epoch time.
-	 *
-	 * @return Time index
-	 */
-	private static long getTimeIndex(AuthenticationContext context) throws TOTPException {
-		return System.currentTimeMillis() / 1000 / TOTPUtil.getTimeStepSize(context);
-	}
+    /**
+     * Get Time steps from unix epoch time.
+     *
+     * @return Time index
+     */
+    private static long getTimeIndex(AuthenticationContext context) throws TOTPException {
+        return System.currentTimeMillis() / 1000 / TOTPUtil.getTimeStepSize(context);
+    }
 
-	/**
-	 * Generate TOTP token for a locally stored user.
-	 *
-	 * @param username Username of the user
-	 * @param context  Authentication context
-	 * @return TOTP token as a String
-	 * @throws TOTPException When could not find user realm for the given tenant domain, invalid
-	 * secret key, decrypting invalid key and could not find the configured hashing algorithm
-	 */
-	public static String generateTOTPTokenLocal(String username, AuthenticationContext context)
```

Comment:
any reason why this public method was removed?

Reply:
This was not removed. TOTPTokenGenerator class was formatted.